src/language-js/print/call-arguments.js

```diff
@@ -71,6 +69,8 @@ function printCallArguments(path, options, print) {
   });
 
   // Dynamic imports cannot have trailing commas
+  const isDynamicImport =
+    node.type === "ImportExpression" || node.callee?.type === "Import";
```

Comment:
The `?` can be removed? Other types except `ImportExpression` should all have `callee`.